docs/reference/mapping/types/parent-join.asciidoc

```diff
@@ -337,88 +348,3 @@ GET _nodes/stats/indices/fielddata?human&fields=my_join_field#question
 --------------------------------------------------
 // CONSOLE
 // TEST[continued]
-
-==== Multiple levels of parent join
-
-It is also possible to define multiple children for a single parent:
-
-[source,js]
---------------------------------------------------
-PUT my_index
-{
-  "mappings": {
-    "doc": {
-      "properties": {
-        "my_join_field": {
-          "type": "join",
-          "relations": {
-            "question": ["answer", "comment"]  <1>
-          }
-        }
-      }
-    }
-  }
-}
---------------------------------------------------
-// CONSOLE
-
-<1> `question` is parent of `answer` and `comment`.
-
```

Comment:
Multiple children does not have an impact on performance so I think we should keep this example in the docs.
The format to allow this is different than the normal one so it should be documented ?